doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/primitives/text/typography/TextTypographyPage.swift

```diff
@@ -0,0 +1,349 @@
+//
+
+import SwiftUI
+
+struct TextTypographyPage: View {
+    var attributedString: AttributedString {
+        var attributedString = AttributedString("Blue text")
+        attributedString.foregroundColor = .blue
+        return attributedString
+    }
+    
+    var days = 5;
+    var closedRangeDate: ClosedRange<Date> {
+        let startDate = Date() // current date and time
+        let endDate = Calendar.current.date(byAdding: .day, value: days, to: startDate)!
+
+            return startDate...endDate
+    }
+    
+    var dateInterval: DateInterval {
+        let startDate = Date()
+        let endDate = Calendar.current.date(byAdding: .day, value: days, to: startDate)!
+        return DateInterval(start: startDate, end: endDate)
+    }
+    
+    var date: Date {
+        let components = DateComponents(year: 2023, month: 12, day: 7)
+        let calendar = Calendar.current
+        return calendar.date(from: components)!
+    }
+    
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 24)  {
+                
```

Comment:
Is there any reason for so many blank lines up here? One blank line is fine, but generally I don't expect to see 3 blank lines.